google/cloud/security/common/gcp_api/storage.py

```diff
@@ -129,3 +129,22 @@ def get_buckets(self, project_id):
             LOGGER.error(api_errors.ApiExecutionError(project_id, e))
             # TODO: pass in "buckets" as resource_name variable
             raise api_errors.ApiExecutionError('buckets', e)
+
+    def get_bucket_acls(self, bucket_name):
+        """Gets acls for GCS bucket.
+
+        Args:
+            bucket_name: The name of the bucket.
+
+        Returns: ACL json for bucket
+
+        """
+        storage_service_api = self.service.bucketAccessControls()
```

Comment:
This could become more simple by..

```
storage_service_api = self.service.bucketAccessControls()
bucket_acl_request = storage_service_api.list(bucket=bucket_name)
try:
    return bucket_acl_request.execute()
except:
....
```